### PR TITLE
HPCC-14964 Simplify filter in FileSpray.DropZoneFileSearch

### PR DESCRIPTION
**Pull request by wangkx**
After discussing with Miguel, we decide to simplify the filters
in FileSpray.DropZoneFileSearch. The new filter will be the same
as Windows Explorer's file search.

Signed-off-by: wangkx kevin.wang@lexisnexis.com

### CODE REVIEW

**Comment by wangkx** [comment]
@RussWhitehead please review.

File: esp/scm/ws_fs.ecm [hunk 1]
```diff
@@ -540,10 +540,7 @@ ESPresponse [exceptions_inline] FileListResponse
 ESPrequest [nil_remove] DropZoneFileSearchRequest
 {
     string DropZoneName;
-    string FileName;
```
Comment (richardkchapman) [inline esp/scm/ws_fs.ecm]: Any compatibility concerns with removing these fields from the interface?

Comment (wangkx) [inline esp/scm/ws_fs.ecm]: This method was added to 6.0.0 several days ago. It has not been used by any one. And we haven't done any build for 6.0.0. I think it should be no compatibility concern.

Comment (richardkchapman) [inline esp/scm/ws_fs.ecm]: Ok, cool, just checking!

**Pull request merged**